packages/api/src/api/helix/stream/HelixStreamApi.ts

```diff
@@ -119,15 +119,9 @@ export class HelixStreamApi extends BaseApi {
 	 * @param user The user ID to retrieve the stream for.
 	 */
 	async getStreamByUserId(user: UserIdResolvable): Promise<HelixStream | null> {
-		const userId = extractUserId(user);
-		const result = await this._client.callApi<HelixPaginatedResponse<HelixStreamData>>({
-			url: 'streams',
-			type: 'helix',
-			userId,
-			query: createStreamQuery({ userName: userId })
-		});
+		const result = await this.getStreamsByUserIds([user]);
```

Comment:
You removed the `userId` association here. Please revert.

Reply:
When this gets passed to `getStreamsByUserIds`, is the user ID not extracted and mapped to the `userId` parameter (see line 111)?

Reply:
Yes, but that does not pass the user ID through to the `callApi` call, which is intentionally overwritten here to allow for sending the request with the explicitly given user's token.

Reply:
So then by that logic, `getStreamByUserName` is currently incorrectly written, correct? Or are you saying that this particular method should be inconsistent with the 3 others above it?

Reply:
By name is different, because it can by design not be used to identify a token.

Reply:
To clarify, the parameter that is relevant is the one in line 126, not the one in line 127.

Reply:
Reverted.